src/components/preferences/routes.ts

```diff
@@ -33,7 +33,15 @@ export const routeGetPrefs = async (request: any, reply: any) => {
   const languageOptions = readLanguageOptions()
   const latestSnapshot = await databaseConnect.getSystemInfo('snapshot')
   const allowedTableNames = config.allowedTableNames
-  reply.send({ preferences: prefs.web, languageOptions, latestSnapshot, allowedTableNames })
+  const logoutAfterInactivity = prefs.server.logoutAfterInactivity ?? DEFAULT_LOGOUT_TIME
+
+  reply.send({
+    preferences: { ...prefs.web, logoutAfterInactivity },
```

Comment:
This preference value is used by both server and client, so we need to pop it in here from the server values.